packages/casl-vue/src/plugin.js

```diff
@@ -15,11 +16,30 @@ export function abilitiesPlugin(Vue, providedAbility) {
   Object.defineProperty(Vue.prototype, '$ability', { value: ability });
 
   Vue.mixin({
+    provide() {
+      return {
+        ability: undefined
+      };
+    },
     methods: {
       $can(...args) {
         watcher.rules = watcher.rules; // create dependency
         return this.$ability.can(...args);
+      },
+      $defineAbility(fn) {
+        const watcherForComponent = new Vue({
+          data: {
+            rules: []
+          }
+        });
+        const abilityForComponent = AbilityBuilder.define(fn);
+        abilityForComponent.on('updated', ({ rules }) => {
+          watcherForComponent.rules = rules;
+        });
+        abilityForComponent.watcherForComponent = watcherForComponent;
+        return abilityForComponent;
       }
     }
   });
+  Vue.component('Can', Can);
```

Comment:
this should be removed and I'd like people (i.e., clients of CASL) to do it manually in case if they want to use `Can` component. In case if not, component can be shaked out from the build by webpack or rollup.

So, `@casl/vue` should just export `Can` component and later in the app if you want to use it you will do this:

```js
import { Can, abilitiesPlugin } from '@casl/vue'
import Vue from 'vue'

Vue.use(abilitiesPlugin)
Vue.component('Can', Can)

new Vue(...)
```